### DISCUSSION TITLE
What is the purpose of "Yank URL" in this prompt?

### DISCUSSION THREAD

**Discussion by emanuele6** [qutebrowser/qutebrowser#6187] [Q&A] [answered]
How to get this prompt:
* run `:download`
* enter `/home/dowload/` (or any path with a trailing `/` to a directory that doesn't exist)

![file](https://user-images.githubusercontent.com/20175435/108591408-e9b9e800-7368-11eb-9779-fd96abdea699.png)

Why is there `Yank URL` as an option? Selecting that option just copies `file:///home/dowload/` to the clipboard without closing the prompt. I was wondering what that is useful for.

Extra: Can we also add an option to go back to the download path prompt? Most times, when I get this prompt, I get it because I spelled incorrectly the name of the directory in which I want to save the file and I have to restart the download to enter the correct path.

**Comment by emanuele6** [comment]
Oh, I figured it out. That option is there just because, that is a "yesno prompt" and all yesno prompts have it .

**Reply by emanuele6** [reply]
How do I convert this "discussion" to a feature request issue?

**Comment by The-Compiler** [comment, marked as answer]
See 520b4733501a274a03482d25a7d5c08dabcacbea - it was added to all prompts where we have a URL available. It probably makes more sense in the download location prompt itself, but it also doesn't hurt to have it there.

As for getting back to the download path prompt, see #2103 which is quite similiar. I added a comment there.
